spec/license_wrap_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'word wrapping' do
+  Dir["#{licenses_path}/*.txt"].each do |file|
+    context "the #{File.basename(file, '.txt')} license" do
+      it 'does not wrap at line length 78' do
+        file_content = File.read(file, encoding: 'utf-8')
+        license_text = file_content.match(/\A(?:---\n.*\n---\n+)?(.*)/m)[1]
+        max_line_length = 0
+        max_line = ''
+        license_text.lines.each do |line|
```

Comment:
I think you can use `max_by` here, e.g., `max_line = license_text.lines.max_by { |line| line.strip.length }`